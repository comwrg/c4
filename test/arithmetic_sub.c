#include <stdio.h>

int main() {
    int g = 88 - 22;
    printf("%d\n", g);
    return 2333;
}
