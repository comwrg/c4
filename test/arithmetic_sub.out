66
exit(2333)
